.bsp/sbt.json

```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.5.6","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/usr/lib/jvm/java-8-oracle/jre/bin/java","-Xms100m","-Xmx100m","-classpath","/usr/share/sbt/bin/sbt-launch.jar","xsbt.boot.Boot","-bsp","--sbt-launch-jar=/usr/share/sbt/bin/sbt-launch.jar"]}
```

Comment:
Delete this file and add .bsp to gitignore please

Reply:
That is done, though my git knowledge is terrible so worth checking the ignore is correct